src/metaschema/readme.md

```diff
@@ -4,23 +4,23 @@ An OSCAL Metaschema defines the the information structures that define an OSCAL
 
 OSCAL Metaschemas are used to generate other OSCAL artifacts based on the metaschema description. These artifacts include:
 
-- XML and JSON Schemas, or *data models*, are derived from the metaschema, expressed as XSD and JSON Schema;
+- XML and JSON schemas, or *data models*, which are derived from the metaschema and expressed as XSD and JSON Schema;
 - Conversion scripts that can convert OSCAL content between the XML and JSON OSCAL formats, ensuring the resulting content is schema valid;
 - XML and JSON [model documentation](https://pages.nist.gov/OSCAL/docs/schemas/) for use on the [OSCAL website](https://pages.nist.gov/OSCAL).
 
-Use of Metaschemas in OSCAL allow us to maintain seamless and consistent support for multiple OSCAL model formats, including XML and JSON. Content can also be kept up-to-date in multiple formats using generated content converters, and can be validated using generated schema. Adding support for new formats (e.g., YAML) can accomplished by extending the Metaschema tooling to produce schema and converters for other formats.
+Use of Metaschemas in OSCAL allows us to maintain seamless and consistent support for multiple OSCAL model formats, including XML and JSON. Content can also be kept up-to-date in multiple formats using generated content converters, and can be validated using generated schemas. Adding support for new formats (e.g., YAML) can accomplished by extending the Metaschema tooling to produce schemas and converters for other formats.
```

Comment:
Blocking: I believe Metaschema maintainers now use the terminology Metaschema modules to be less ambiguous and confusing when talking about Metaschema (MIMF, aka the Metaschema Information Modeling Framework with the more formal title) versus "metaschemas" now being Metaschema modules (one or more model documents defined using MIMF), I would think we should align on that. Additionally, we seem to be repeating a lot of information that should be on the Metaschema site, so it makes more sense to point them there I think.

```suggestion
Use of Metaschema modules in OSCAL allows us to maintain seamless and consistent support for multiple OSCAL model formats, including XML and JSON. More information can be found at the [Metaschema Information Modeling Framework documentation site](https://pages.nist.gov/metaschema/)
```

Reply:
I updated the terminology and included the link, in 11dfbb82a7f00973a0cfd911911a0a2ecc27214e.